src/frontend/src/common/components/library/data_subview/components/DeleteSamplesConfirmationModal/index.tsx

```diff
@@ -39,14 +40,13 @@ const DeleteSamplesConfirmationModal = ({
   });
 
   const onDelete = () => {
+    setNumDeletedSamples(samplesToDelete.length);
     deleteSampleMutation.mutate({
       samplesToDelete,
     });
```

Comment:
This is a bit unnecessary wishlist-y, but if we're running `setNumDeletedSamples` in the block where the deletion actually takes place, can we just scoop up the delete API response and use its length? ... Looking at backend code now ... For bulk deletion, that's easy. Looks like deleting a single one just returns the id as an int though, so there might be some special-casing between bulk and singles, not sure it's worth the hassle. Although wait, do we ever call the single deletion endpoint from FE, or do we always use bulk and just pass it a 1-length array?

Reply:
To be clear, the reason why I think this is a good idea is in case some deletions don't go through, we have a way of knowing. Otherwise we could say we deleted 47, but only 46 deletions went through or something. That said, looking at current BE implementation, it appears that if any of the requested samples for deletion won't get deleted, the whole thing just 404s instead of deleting at all, so my worry can't actually come to pass currently.

Reply:
Currently we don't call single deletes for samples so special casing wouldn't be necessary.